servicediscovery/requests.go

```diff
@@ -0,0 +1,19 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package servicediscovery
+
+const DefaultNamespace = "default"
+
+type ResolveRequest struct {
+	ID        string
+	Namespace string
+	Port      int
+	Data      map[string]string
+}
+
+func NewResolveRequest() *ResolveRequest {
+	return &ResolveRequest{Namespace: "default"}
```

Comment:
nit: you might want to use DefaultNamespace const instead of "default"

Reply:
It's meant to be 'default' like the k8s default namespace.

Reply:
You defined `DefaultNamespace` const in this code. why not use that const?

Reply:
Oh, right.  Didn't replace, done.